release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2905,6 +2905,8 @@ To avoid this issue, you can enable confidential VMs on an existing cluster by u
 
 * Installation fails when installing {product-title} with the `bootMode` set to `UEFISecureBoot` on a node where Secure Boot is disabled. Subsequent attempts to install {product-title} with Secure Boot enabled will proceed normally. (link:https://issues.redhat.com/browse/OCPBUGS-19884)[*OCPBUGS-19884*])
 
+* In {product-title} {product-version}, a `MachineConfig` with Ignition version 3.4 might fail `ocp4-pci-dss` scans of the `api-collector` pods with `CrashLoopBackOff` errors, causing the Compliance Operator to not work as expected. (link:https://issues.redhat.com/browse/OCPBUGS-18025[*OCPBUGS-18025*])
```

Comment:
Maybe remove `ocp4-pci-dss`? actually not this profiles is impacted. cis, pci-dss,moderate.
Another minor issue about `In OpenShift Container Platform Branch Build`, is it expected? Thanks.
`In OpenShift Container Platform Branch Build, a MachineConfig with Ignition version 3.4 might fail ocp4-pci-dss scans of the api-collector pods with CrashLoopBackOff errors, causing the Compliance Operator to not work as expected. ([OCPBUGS-18025](https://issues.redhat.com/browse/OCPBUGS-18025))`

Reply:
All others looks good to me.

Reply:
I removed the `ocp4-pci-dss`. As for the `{product-version}` rendering as Branch Build, which is expected for the preview. When displayed in 4.14, it will say OpenShift Container Platform 4.14.

Now it reads:
```In OpenShift Container Platform Branch Build, a MachineConfig with Ignition version 3.4 might fail scans of the api-collector pods with CrashLoopBackOff errors, causing the Compliance Operator to not work as expected. ([OCPBUGS-18025](https://issues.redhat.com/browse/OCPBUGS-18025))